relay-server/src/metrics_extraction/spans/mod.rs

```diff
@@ -299,10 +301,31 @@ fn sanitized_span_description(
         sanitized.push_str(&format!("{domain}/"));
     }
     sanitized.push_str("<unparameterized>");
+    sanitized = truncate_span_description(sanitized);
 
     Some(sanitized)
 }
 
+/// Trims the given span description to ensure
+/// [`relay-metrics::AggregatorService`] doesn't drop it for being too long.
+///
+/// If the description is short, it remains unchanged. If it's long, this method
+/// truncates it to the maximum allowed length and sets the last character to
+/// `*`. The maximum length is determined by [`MaxChars::TagValue`].
+///
+/// Note [`relay-metrics::AggregatorService`] is configured independently and
+/// doesn't use this type; this method uses it for simplicity, and because at
+/// the time of writing this the values are the same. If the limit this type
+/// provides is longer, this method may be ineffective.
+fn truncate_span_description(description: String) -> String {
+    let mut truncated = description;
+    if bytecount::num_chars(truncated.as_bytes()) > MaxChars::TagValue.limit() {
+        truncated.truncate(MaxChars::TagValue.limit() - 1);
```

Comment:
Using `MaxChars` is definitely not ideal here (as mentioned in the docstrings above), but I think it's better than hardcoding a value here. I believe building a proper solution to share the config with `AggregatorService` is out of scope for now.